prepare_GBM_data.R

```diff
@@ -0,0 +1,195 @@
+# Script prepares GBM expression data for use in pipelines
+# For GBM array data: convert sample names and remove duplicate individuals
+# For GBM seq data: filter all seq data for GBM samples only, convert genes IDs
+
+# Steven Foltz July 2021
+
+option_list <- list(
+  optparse::make_option("--seq_input",
+                        default = NULL,
+                        help = "TCGA sequencing expression input file path"),
+  optparse::make_option("--array_input",
+                        default = NULL,
+                        help = "refine.bio microarray expression input file path"),
+  optparse::make_option("--metadata_input",
+                        default = NULL,
+                        help = "refine.bio aggregated metadata JSON file path"),
+  optparse::make_option("--array_output",
+                        default = NULL,
+                        help = "Processed microarray data output file path"),
+  optparse::make_option("--seq_output",
+                        default = NULL,
+                        help = "Processed sequencing data output file path"),
+  optparse::make_option("--clinical_input",
+                        default = NULL,
+                        help = "Clinical information input file path (Excel file)"),
+  optparse::make_option("--clinical_output",
+                        default = NULL,
+                        help = "Clinical information output file path (.tsv)"),
+  optparse::make_option("--overwrite",
+                        action = "store_true",
+                        default = FALSE,
+                        help = "Overwrite existing output files [default: %default]")
+)
+
+opt <- optparse::parse_args(optparse::OptionParser(option_list=option_list))
+source("util/option_functions.R")
+check_options(opt, "prepare_GBM_data.R") # how to get this script name easily?
```

Comment:
To me, it seems better to not have the name of the script where called included as one of the arguments. It's only used for messages as far as I can tell and if you're concerned about informative messages while running the shell script, I think you could probably could spit out some text in the shell script right before you run this step?

Reply:
Yes that makes sense to me. So rely on the shell script for knowing where we are in the process, and the output from `check_options()` for what happened.